From patchwork Mon Jun 20 13:14:55 2022
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Ross Burton <ross.burton@arm.com>
X-Patchwork-Id: 9386
Return-Path: <ross.burton@arm.com>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from aws-us-west-2-korg-lkml-1.web.codeaurora.org
 (localhost.localdomain [127.0.0.1])
	by smtp.lore.kernel.org (Postfix) with ESMTP id B5D3DCCA486
	for <webhook@archiver.kernel.org>; Mon, 20 Jun 2022 13:15:07 +0000 (UTC)
Received: from foss.arm.com (foss.arm.com [217.140.110.172])
 by mx.groups.io with SMTP id smtpd.web09.30183.1655730901322106598
 for <openembedded-core@lists.openembedded.org>;
 Mon, 20 Jun 2022 06:15:02 -0700
Authentication-Results: mx.groups.io;
 dkim=missing;
 spf=pass (domain: arm.com, ip: 217.140.110.172,
 mailfrom: ross.burton@arm.com)
Received: from usa-sjc-imap-foss1.foss.arm.com (unknown [10.121.207.14])
	by usa-sjc-mx-foss1.foss.arm.com (Postfix) with ESMTP id A1EE6113E;
	Mon, 20 Jun 2022 06:15:00 -0700 (PDT)
Received: from oss-tx204.lab.cambridge.arm.com
 (usa-sjc-imap-foss1.foss.arm.com [10.121.207.14])
	by usa-sjc-imap-foss1.foss.arm.com (Postfix) with ESMTPSA id 0DBE43F792;
	Mon, 20 Jun 2022 06:14:59 -0700 (PDT)
From: Ross Burton <ross.burton@arm.com>
To: openembedded-core@lists.openembedded.org
Cc: nd@arm.com
Subject: [PATCH v2 1/4] Add python3-pathspec (from meta-python)
Date: Mon, 20 Jun 2022 14:14:55 +0100
Message-Id: <20220620131458.4177628-1-ross.burton@arm.com>
X-Mailer: git-send-email 2.25.1
MIME-Version: 1.0
List-Id: <openembedded-core.lists.openembedded.org>
X-Webhook-Received: from li982-79.members.linode.com [45.33.32.79] by
 aws-us-west-2-korg-lkml-1.web.codeaurora.org with HTTPS for
 <openembedded-core@lists.openembedded.org>; Mon, 20 Jun 2022 13:15:07 -0000
X-Groupsio-URL: 
 https://lists.openembedded.org/g/openembedded-core/message/167101

This is a dependency of the Hatch build system, to be added as
python3-hatchling.

Signed-off-by: Ross Burton <ross.burton@arm.com>

Backported to kirkstone

Signed-off-by: Jasper Orschulko <Jasper.Orschulko@iris-sensing.com>
---
 .../recipes-devtools/python/python3-pathspec_0.9.0.bb | 11 +++++++++++
 1 files changed, 12 insertions(+)
 create mode 100644 meta/recipes-devtools/python/python3-pathspec_0.9.0.bb

diff --git a/meta/recipes-devtools/python/python3-pathspec_0.9.0.bb b/meta/recipes-devtools/python/python3-pathspec_0.9.0.bb
new file mode 100644
index 00000000000..641f1cfd87c
--- /dev/null
+++ b/meta/recipes-devtools/python/python3-pathspec_0.9.0.bb
@@ -0,0 +1,11 @@
+SUMMARY = "Utility library for gitignore style pattern matching of file paths."
+HOMEPAGE = "https://github.com/cpburnz/python-path-specification"
+SECTION = "devel/python"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+SRC_URI[sha256sum] = "e564499435a2673d586f6b2130bb5b95f04a3ba06f81b8f895b651a3c76aabb1"
+
+inherit pypi setuptools3
+
+BBCLASSEXTEND = "native nativesdk"
